examples/docking-protein-protein/docking-protein-protein.cfg

```diff
@@ -4,7 +4,7 @@
 # directory in which the scoring will be done
 run_dir = "run1"
 ncores = 20
-# cns_exec = "path/to/bin/cns" # optional
+cns_exec = "/Users/abonvin/haddock3/bin/cns"
```

Comment:
@amjjbonvin  Care adding these paths. Use `cns_exec = ""` will read the `bin` installation by default.